### ISSUE TITLE
Loans: Add reverse rebalancing

### ISSUE THREAD

**Issue by BennyOptions** [balancednetwork/balanced-contracts#239]
https://docs.google.com/document/d/17r9ephKwyuHP64WDR2qdLwtJi8JXfydxrVT7_TnwhpQ/edit

**Comment by BennyOptions** [comment]
This is blocked by another issue related to assessing rebalancing against all users instead of batches of users

**Comment by BennyOptions** [comment]
Take the minimum of two numbers:

min(amountNeededToCorrect, 0.05% of group's debt)

**Comment by BennyOptions** [comment]
We should consider having two separate lists for Reverse Rebalancing vs normal rebalancing or two cursor/index, but this could introduce other issues.
